### PR TITLE
Enhance UI for the user-profile section

### LINKED ISSUE #2591: UI for the user profile page
**Is your feature request related to a problem? Please describe.**

The UI seems to be very cohesive between the image upload instruction and User name.

**Describe the solution you'd like**

- To minimize the image upload instruction which introduces a line break between.

**Describe alternatives you've considered**

- To enchance the UI for user name ,family name and email.
 
**Screenshot**
Screenshot 2019-04-12 02:24:07

**Additional context**

Share views, like to work on it

### PR DESCRIPTION
**Pull request by mohitm15**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Enhances the UI by the first approach discussed in the issue.

#### Changes proposed in this pull request:

- update `user-profile.hbs`

#### Screenshots
Screenshot 2019-04-12 02:43:53

Fixes #2591

### CODE REVIEW

File: app/templates/components/forms/user-profile-form.hbs [hunk 1]
```diff
@@ -11,6 +11,7 @@
         maxSizeInKb=10000
         aspectRatio=(array 1 1)
         helpText=(t 'We recommend using at least a 1000x1000px (1:1 ratio) image')}}
+      <br>
```
Comment (CosmicCoder96) [inline app/templates/components/forms/user-profile-form.hbs]: use `ui hidden divider`

**Comment by kushthedude** [comment]
@mohitm15 I have observed usually in your PR , you always use your development branch to create PR and then you delete it . Like in this PR you deleted your development and here it shows `UNKNOWN REPOSITORY` it is not a good practice , as if mentors want you to change something you cant change and you need to open a new PR . This is not a good practice to follow , Please change this

File: app/templates/components/forms/user-profile-form.hbs (hunk 1 above)
Comment (kushthedude) [reply app/templates/components/forms/user-profile-form.hbs]: @mohitm15  I have observed usually in your PR , you always use your development branch to create PR and then you delete it . Like in this PR you deleted your development and here it shows `UNKNOWN REPOSITORY` it is not a good practice , as if mentors want you to change something you cant change and you need to open a new PR . This is not a good practice to follow , Please change this , Else its your wish , No problem . 
Now see you cant change your commits which @CosmicCoder96 is asking for you to do

**Comment by mohitm15** [comment]
> @mohitm15 I have observed usually in your PR , you always use your development branch to create PR and then you delete it . Like in this PR you deleted your development and here it shows `UNKNOWN REPOSITORY` it is not a good practice , as if mentors want you to change something you cant change and you need to open a new PR . This is not a good practice to follow , Please change this

@kushthedude Ya sure,  just revising again the git tutorial.Take care from next time.

**Comment by kushthedude** [comment]
@mohitm15 Revise the tut before implementing , Most of your PR are merged from a unknown repository . Please take care for same

**Comment by mohitm15** [comment]
@CosmicCoder96 Should I close this PR and make another as this is from an unknown branch

File: app/templates/components/forms/user-profile-form.hbs (hunk 1 above)
Comment (mohitm15) [reply app/templates/components/forms/user-profile-form.hbs]: @CosmicCoder96 Should I close this PR and open a new as this is from unknown branch and I am unable to make changes in it.

**Comment by CosmicCoder96** [comment]
Yes.

**Comment by mohitm15** [comment]
@CosmicCoder96 I had revisited git now. will not repeat this practice.

**Pull request closed without merging**